fe2-android/app/src/test/ui/robolectric/com/github/dedis/popstellar/ui/lao/socialmedia/ChirpListAdapterTest.kt

```diff
@@ -301,24 +303,74 @@ class ChirpListAdapterTest {
     }
   }
 
+  @Test
+  fun profileDisplayTest() {
+    val mockSocialMediaViewModel = Mockito.mock(SocialMediaViewModel::class.java)
+    Mockito.`when`(mockSocialMediaViewModel.chirps)
+            .thenReturn(Observable.just(ArrayList(listOf(CHIRP_1, CHIRP_3))))
+    Mockito.`when`(mockSocialMediaViewModel.getReactions(LAO_ID, MESSAGE_ID_1))
+            .thenReturn(Observable.just(setOf(REACTION)))
+    Mockito.`when`(mockSocialMediaViewModel.getReactions(LAO_ID_2, MESSAGE_ID_3))
+            .thenReturn(Observable.just(setOf(REACTION2)))
+
+    val chirps = ArrayList(listOf(CHIRP_1, CHIRP_3))
+    activityScenarioRule.scenario.onActivity { activity: LaoActivity ->
+      val socialMediaViewModel = mockSocialMediaViewModel
+      val viewModel = obtainViewModel(activity)
+      val chirpListAdapter =
+              createChirpListAdapter(activity, viewModel, socialMediaViewModel, chirps)
+
+      // Use a non null ViewGroup to inflate the card
+      val layout = LinearLayout(activity.applicationContext)
+      val parent = TextView(activity.applicationContext)
+      parent.text = "Mock Title"
+      layout.addView(parent)
+
+      // Get the view for the first chirp in the list.
+      val view1 = chirpListAdapter.getView(0, null, layout)
+      Assert.assertNotNull(view1)
+
+      // The chirp of our LAO should have an unmodified color (null in this case)
+      val profile1 = view1.findViewById<ImageView>(R.id.social_media_profile)
+      Assert.assertNotNull(profile1)
+      Assert.assertNull(profile1.imageTintList)
+
+      // Get the view for the second chirp in the list.
+      val view2 = chirpListAdapter.getView(1, null, layout)
+      Assert.assertNotNull(view2)
+
+      // The chirp of the other LAO should be gray
+      val profile2 = view2.findViewById<ImageView>(R.id.social_media_profile)
+      Assert.assertNotNull(profile2)
+      Assert.assertEquals(-8355712, profile2.imageTintList?.defaultColor)
```

Comment:
Would it be possible to use the android color resource here instead of hardcoding the value?

Reply:
Yes, it will be cleaner